go/mounter/keybase-mount-helper/main.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2018 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// +build !windows
+
+// keybase-mount-helper is a simple program intended to be run with
+// SUID permissions as a keybase helper user, which manages the
+// /var/lib/keybase/mount1 symlink, pointing to a user's KBFS
+// mountpoint.  The first user to run this program captures the
+// symlink; others will get an error.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/user"
+	"runtime"
+	"strconv"
+	"syscall"
+)
+
+const (
+	kbUsername = "keybasehelper"
+)
+
+var (
+	fFirstMountLink           string
+	fOriginalFirstMountTarget string
+	fUserMount                string
+)
+
+func getDefaultUserMount() string {
+	// On linux, `run_keybase` passes in an explicit mountpoint in
+	// ${XDG_DATA_HOME:-$HOME/.local/share}, so in practice the
+	// default here probably isn't used.
+	return fmt.Sprintf("/run/user/%d/keybase/fs", os.Getuid())
```

Comment:
It's a little weird to hardcode `/run/user/...`, since it's almost impossible for that to be a valid path unless `XDG_RUNTIME_DIR` is also set to refer to it. Maybe we should either use `XDG_RUNTIME_DIR` here, or just hardcode the expected `$HOME/.local/share` thing? Though I guess this is mostly academic, like you mention in your comment.

Reply:
Ok, you guys win.  I'll implement a little function that basically does the same thing as `libkb.Env.GetMountDir()`, without actually pulling in `libkb` since that would cause huge binary bloat.